### ISSUE TITLE
哈喽

### ISSUE THREAD

**Issue by tang-zhonghua** [ludlows/CSI-Activity-Recognition#1]
哈喽    有你这篇论文链接吗

**Comment by ludlows** [comment]
没有，如果你有兴趣发表，请带个我的名字

**Comment by tang-zhonghua** [comment]
老哥   attention那里prob_kernel是什么作用啊   一般不都是经过打分函数之后，直接由softmax得到权重的吗？    还是说 atten_state 是打分函数 ，logits是得分啊
atten_state = tf.tanh(tf.tensordot(input_tensor, self.kernel, axes=1) + self.bias) 
logits = tf.tensordot(atten_state, self.prob_kernel, axes=1)                    
prob = tf.nn.softmax(logits)  
 新手，老哥指点下，谢了
